modules/affile/affile-grammar.ym

```diff
@@ -260,10 +260,6 @@ multi_line_option
 	    CHECK_ERROR_GERROR(log_proto_multi_line_server_options_set_garbage(&last_log_proto_options->super, $3, &error), @3, error, "error compiling multi-line regexp");
 	    free($3);
 	  }
-	| KW_MULTI_LINE_TIMEOUT '(' nonnegative_integer ')'
```

Comment:
This is a good approach, but not quite the ideal one, it has one problem:
Removing `KW_MULTI_LINE_TIMEOUT` from `multi_line_option` removes it from other users of `multi_line_option`, like `source_affile_option`.

Please only remove it from `source_afpipe_option`.